docusaurus/docs/develop/developer_guide/adding_params.md

```diff
@@ -43,245 +50,468 @@ https://github.com/ignite/cli/issues/3684#issuecomment-2299796210
 
 ## Step-by-Step Instructions
 
+:::warning
+The steps outlined below follow **the same example** where:
+- **Module name**: `examplemod`
+- **New parameter name**: `new_param`
+- **Default value**: `42`
+
+When following these steps, be sure to substitute these example values with your own!
+:::
+
+### 0. If the Module Doesn't Already Support a `MsgUpdateParam` Message
+
+In order to support **individual parameter updates**, the module MUST have a `MsgUpdateParam` message.
+If the module doesn't already support this message, it will need to be added.
+
+#### 0.1 Scaffold the `MsgUpdateParam` Message
+
+Use `ignite` to scaffold a new `MsgUpdateParam` message for the module.
+Additional flags are used for convenience:
+
+```bash
+ignite scaffold message update-param --module examplemod --signer authority name as_type --response params
+```
+
```

Comment:
```suggestion

:::note
If you experience errors like these:

\```
✘ Error while running command /home/bwhite/go/bin/buf generate /tmp/proto-sdk2893110128/cosmos/upgrade/v1beta1/tx.proto ...: {..."message":"import \"gogoproto/gogo.proto\": file does not exist"}
: exit status 100
\```

\```
✘ Error while running command go mod tidy: go: 
...                                                                                              
go: github.com/pokt-network/poktroll/api/poktroll/examplemod imports                                                                                   
        cosmossdk.io/api/poktroll/shared: module cosmossdk.io/api@latest found (v0.7.6), but does not contain package cosmossdk.io/api/poktroll/shared
\```

Then try running `make proto_clean_pulsar`.
:::

```